Dockerfile-bionic.template

```diff
@@ -0,0 +1,17 @@
+ARG BASE_IMAGE=rstudio/r-base
+FROM ${BASE_IMAGE}:%%VARIANT%%
+
+ARG R_VERSION=%%R_VERSION%%
+ARG OS_IDENTIFIER=%%OS_IDENTIFIER%%
+
+# Install R
+RUN wget https://cdn.rstudio.com/r/${OS_IDENTIFIER}/pkgs/r-${R_VERSION}_1_amd64.deb && \
+    apt-get update -qq && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -f -y --no-install-recommends ./r-${R_VERSION}_1_amd64.deb && \
```

Comment:
There might be a better way to install from a local deb without gdebi, but this is what I found from googling: `apt-get install -f /path/to-deb`. 

The `--no-install-recommends` is so we don't bring in `x11-utils` from `xdg-utils`'s Recommends, which is quite heavy (~220M)